<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(219, 222, 222);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="hostEdit_">
       <property name="styleSheet">
        <string notr="true">font: 25 italic 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>IP Address :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostEdit">
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="text">
        <string>192.168.1.106</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="portEdit_">
       <property name="styleSheet">
        <string notr="true">font: 25 italic 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>Port : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit">
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="text">
        <string>15020</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="battery_">
       <property name="styleSheet">
        <string notr="true">border: none;
font: 25 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Battery&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="battery">
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(224, 224, 224);</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="batteryIcon">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="logoEsirem">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>211</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressources.qrc">
      <normaloff>:/img/esirem/logo_Esirem.png</normaloff>:/img/esirem/logo_Esirem.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="gridFrame">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(44, 162, 189);</string>
    </property>
    <layout class="QGridLayout" name="SensorsLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="Sensors_">
       <property name="styleSheet">
        <string notr="true">border: none;
font: 75 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Sensors&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="speedL_">
       <property name="styleSheet">
        <string notr="true">border: none;
font: 25 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Left Speed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="odoL_">
       <property name="styleSheet">
        <string notr="true">border: none;
font: 25 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Odometer Left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLCDNumber" name="speedR">
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(183, 186, 186);
background: rgb(244, 247, 247);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLCDNumber" name="odoL">
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(183, 186, 186);
background: rgb(244, 247, 247);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLCDNumber" name="odoR">
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(183, 186, 186);
background: rgb(244, 247, 247);</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLCDNumber" name="speedL">
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(183, 186, 186);
background: rgb(244, 247, 247);</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="speedR_">
       <property name="styleSheet">
        <string notr="true">border: none;
font: 25 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Right Speed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="odoR_">
       <property name="styleSheet">
        <string notr="true">font: 25 13pt &quot;Helvetica&quot;;
border: none;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Odometer Right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;Helvetica&quot;;
border: none;</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica'; font-size:13pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#1a4456;&quot;&gt;Wifibot&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a90003;&quot;&gt;V&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#1a4456;&quot;&gt;incAnto&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>353</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="speed_">
       <property name="styleSheet">
        <string notr="true">font: 25 13pt &quot;Helvetica&quot;;
border: none;</string>
       </property>
       <property name="text">
        <string>Speed Control : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="speedSlider">
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>75</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="speedLCD">
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="intValue" stdset="0">
        <number>75</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>286</width>
      <height>216</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6"/>
   </widget>
   <widget class="QGroupBox" name="directionBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Direction</string>
    </property>
    <widget class="QPushButton" name="boost">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>211</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Boost !</string>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/img/boost/boost2.svg</normaloff>:/img/boost/boost2.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>353</width>
       <height>304</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="turnAwayButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsMove/undo.svg</normaloff>:/img/directionsMove/undo.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="backwardButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsMove/down.png</normaloff>:/img/directionsMove/down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="leftButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsMove/left.png</normaloff>:/img/directionsMove/left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="forwardButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsMove/up.png</normaloff>:/img/directionsMove/up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="rightButton">
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsMove/right.png</normaloff>:/img/directionsMove/right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="camResetButton_">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>670</y>
      <width>85</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Camera</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="cameraBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>379</width>
       <height>367</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QPushButton" name="camResetButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsCam/boussole2.png</normaloff>:/img/directionsCam/boussole2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="camLeftButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsCam/square.png</normaloff>:/img/directionsCam/square.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="camUpButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsCam/triangle.png</normaloff>:/img/directionsCam/triangle.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="camDownButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsCam/x.png</normaloff>:/img/directionsCam/x.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="camRightButton">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img/directionsCam/circle.png</normaloff>:/img/directionsCam/circle.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QProgressBar" name="speedBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="filtreBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Filtre</string>
    </property>
    <widget class="QPushButton" name="filter1Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Gray</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter2Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Negative</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter3Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Sepia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetFilterButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Reset Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter4Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Blur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter5Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Saturate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter6Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Hue-rotate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter7Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(215, 218, 218);</string>
     </property>
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="capture">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>620</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string>capture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
